spec/requests/patients_request_spec.rb

```diff
@@ -0,0 +1,156 @@
+require 'rails_helper'
+
+# Define patient controller action tests.
+RSpec.describe "Patients", type: :request do
+  # Test index action.
+  context 'GET #index' do
+    it 'should fail get index when not singed in' do
```

Comment:
You correctly followed the standard, but that standard was designed for mintiest, not Rspec. Maybe you can propose a new pattern. Rspec chains together the hierarchical block names, so this would be.
```
GET #index should fail get index ...
```